src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1283,6 +1284,20 @@ namespace DataSurfaces {
         }
     };
 
+    struct SurfaceIncidentSolarMultiplier
+    {
+        // Members
+        std::string Name;
+        int SurfaceIdx; // surface index
+        Real64 Scaler;
```

Comment:
`Multiplier`?

Reply:
I was trying to distinguish the single-value multiplier and the schedule multiplier, both are incident solar multipliers.